ci-operator/step-registry/openshift/selfupgrade/aws/loki/openshift-selfupgrade-aws-loki-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-selfupgrade-aws-loki
+  steps:
+    pre:
+    - ref: ipi-install-monitoringpvc
+    - ref: ipi-install-loki
+    - chain: ipi-aws-pre-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: gather-loki
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:latest"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:initial"
+    env:
+      TEST_COMMAND: "run-upgrade"
+      TEST_SUITE: "all"
+      TEST_OPTIONS: "abort-at=100"
+  documentation: |-
+    The Openshift E2E AWS workflow executes the self-upgrade end-to-end test suite on AWS with a default cluster configuration and loki as log collector, exercising an upgrade during the process.
```

Comment:
I'm not quite certain what does "self-upgrade" means in this context

Reply:
Starting the upgrade from this version, instead of upgrading to it. I should phase it explicitly there.

Reply:
`openshift-upgrade-aws-loki` would do the same - start from `release:initial` (in PRs - current stable) and upgrade to `release:latest` (in PRs - release with suggested change)